src/mode.js

```diff
@@ -65,11 +65,14 @@ export function setModeForTesting(m) {
  * @return {!ModeDef}
  */
 function getMode_() {
+  const OVERRIDE_IS_NOT_PROD = true;
```

Comment:
haha i intentionally made the name non intuitive to not collide with anything in the future lol, will fix